alluxio/configuration.rst

```diff
@@ -0,0 +1,32 @@
+**********************
+Alluxio Configuration
+**********************
+
+
+|fdd| provides Alluxio with the following configurations:
```

Comment:
An indication on where the configuration file is stored, could help better user to check his/her current configuration.